packages/react-ui/src/ui/core/model/api/smart-messages.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  CreateSmartMessageTransactionCommandDto,
+  SmartMessageTransactionDto,
+  SubmitSmartMessageTransactionCommandDto,
+} from './smart-messages.types';
+
+function createSmartMessageApi() {
+  return {
+    cancelSmartMessage: async (
+      dialectCloudUrl: string,
+      smartMessageId: string,
+    ) => {
```

Comment:
ideally, yes, but we don't have access to the dialect cloud url on the lvl of initialization, which is why we need just pass it down as is or rethink how and when we call `createSmartMessageApi`. Again, this is somewhat temporary, since we will be making this api public sometime in the future and it will happen on the sdk lvl, rather than here.